modules/supported-platforms-for-managed-clusters.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.adoc
+
+:_content-type: PROCEDURE
+[id="supported-platforms-for-managed-clusters_{context}"]
+= supported platforms for managed clusters
+
+You can manually specify the platform type for managed clusters if you do not want to set the platform type automatically.
+
+The {ai-full} supports the following platform types:
```

Comment:
It doesn't seem like this list is rendering correctly in the doc preview

Reply:
I think there needs to be a blank line between the intro sentence and the first bullet

Reply:
Updated as suggested.